src/runtime/dom-extras.ts

```diff
@@ -84,10 +84,14 @@ export const patchSlotAppendChild = (HostElementPrototype: any) => {
     if (slotNode) {
       const slotChildNodes = getHostSlotChildNodes(slotNode, slotName);
       const appendAfter = slotChildNodes[slotChildNodes.length - 1];
-      appendAfter.parentNode.insertBefore(newChild, appendAfter.nextSibling);
+      const insertedNode = appendAfter.parentNode.insertBefore(newChild, appendAfter.nextSibling);
+
       // Check if there is fallback content that should be hidden
       updateFallbackSlotVisibility(this);
-      return;
+      // Fore a re-render of the host element
+      forceUpdate(this);
```

Comment:
Can you help me understand why we need to do force an update here?

Reply:
Note: I want a little additional clarification here, after reading the git commit msg

Reply:
I ran into an issue when testing this out with Framework screenshot tests. For some components that had `appendChild` patched, the component would include the append child node after the call until something triggered a re-render of the component (this was as simple as hovering the component). This made it so the component would immediately re-render to reflect the changes and get the tests to pass